src/sidebar/store/modules/groups.js

```diff
@@ -118,6 +119,43 @@ function getGroup(state, id) {
   return state.groups.find(g => g.id === id);
 }
 
+/**
+ * Return groups that don't show up in Featured and My Groups.
+ *
+ * @return {Group|undefined}
+ */
+function getCurrentlyViewingGroups(state) {
```

Comment:
We chatted with voices: It may be useful to experiment with writing this code as an expression of the other collections, e.g. "Current Viewing should be a collection of all groups that are neither in My Groups nor in Featurex Groups", and that way you can conflate the logic for both logged-in and logged-out users.